src/styles/inits/variables.scss

```diff
@@ -0,0 +1,10 @@
+$link-hover-color: #00acdc;
+$border-color: #f3f3f3;
+$main-text-color: #060b35;
+$secondary-text-color: #616070;
+$small-font-size: 10px;
+$medium-font-size: 12px;
+$large-font-size: 16px;
+$default-button-color: #00acdc;
```

Comment:
Don't need to duplicate variables with the same values
<img width="258" alt="image" src="https://github.com/user-attachments/assets/29956922-e2d6-4f92-8f15-ad481b4afc2f" />
```suggestion
```